src/basicfuns.jl

```diff
@@ -224,17 +224,20 @@ logsubexp(x::Real, y::Real) = max(x, y) + log1mexp(-abs(x - y))
 """
     logsumexp(X)
 
-Compute `log(sum(exp, X))`, evaluated avoiding intermediate overflow/undeflow.
+Compute `log(sum(exp, X))` in a numerically stable way that avoids intermediate over- and
+underflow.
 
 `X` should be an iterator of real numbers.
+
+See also: [`logsumexp_onepass`](@ref)
```

Comment:
Move all documentation relevant for end users to this docstring, and remove the one for 
`logsumexp_onepass`, which is internal and not supposed to be used directly.
```suggestion
```